public/offline.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>You are offline</title>
+    <style>
+      body {
+        font-family: helvetica, arial, sans-serif;
+        margin: 2em;
+      }
+
+      h1 {
+        font-style: italic;
+        color: #373fff;
+      }
+      p {
+        margin-block: 1rem;
+      }
+      
+      button {
+        display: block;
+      }
+    </style>    
```

Comment:
refactor into a separate css/scss and then import here

Reply:
@shehzad-haroon is this resolved in this PR?

Reply:
Solved